src/main/kotlin/snyk/amplitude/api/AmplitudeExperimentApiClient.kt

```diff
@@ -0,0 +1,81 @@
+package snyk.amplitude.api
+
+import com.intellij.openapi.diagnostic.logger
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import snyk.net.HttpClient
+import java.io.IOException
+
+class AmplitudeExperimentApiClient private constructor(
+    private val baseUrl: String,
+    private val httpClient: OkHttpClient
+) {
```

Comment:
This is our 3rd ApiClient already :( May be we need abstract/generic ApiClient class to inherite from?

Reply:
Let's sync offline about it.

Reply:
Will be replaced with Experiment Kotlin SDK, so ok for now